Jenkinsfile.content

```diff
@@ -9,41 +9,13 @@ node('vetsgov-general-purpose') {
 
   refStatusState = getGithubCommitState("vets-website", ref)
   if ("${refStatusState}" != "SUCCESS") {
-    slackSend(message:"Web content refresh on ${params.env} aborted due to '${refStatusState}' status on vets-website commit ${ref}.",
+    message = "Web content refresh on ${params.env} aborted due to '${refStatusState}' status on vets-website commit ${ref}."
+    echo message
+    slackSend(message: message,
               color: "warning",
               channel: "cms-notifications")
 
+    currentBuild.result = 'FAILURE'
     return
   }
-
-  dir("vets-website") {
-    checkout scm: [$class: 'GitSCM', branches: [[name: ref]], userRemoteConfigs: [[credentialsId: 'va-bot', url: 'git@github.com:department-of-veterans-affairs/vets-website.git']]]
-  }
-
```

Comment:
Did you mean to delete this section?

Reply:
No, I didn't mean to delete that. I deleted it for testing and copy/pasted into Jenkins. Thanks!

Reply:
Fixed, rebased, force pushed.